src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -97,15 +115,15 @@ class AnonymousElectorDocument(
     var version: Long? = null
 ) {
 
-    val status: AedPrintRequestStatus.Status?
-        get() = getLatestPrintRequest().status
+    val status: AnonymousElectorDocumentStatus.Status?
+        get() = getLatestStatus().status
 
-    fun addPrintRequest(newPrintRequest: AedPrintRequest): AnonymousElectorDocument {
-        printRequests += newPrintRequest
+    fun addStatus(newStatus: AnonymousElectorDocumentStatus): AnonymousElectorDocument {
+        statusHistory += newStatus
         return this
```

Comment:
Out of interest, why does it return `this`?

Reply:
Following same pattern as in `TemporaryCertificate` and `Certificate` - useful when using some of the Kotlin scope functions.